pages/common/dfc.md

```diff
@@ -0,0 +1,20 @@
+# dfc
```

Comment:
Does the command work in windows and macOS?

Reply:
dfc is available on macOS through [brew](https://formulae.brew.sh/formula/dfc)

I would assume it works under WSL but I'm fairly certain its not available for windows outside of that.

Reply:
Nice! In that case, it's in the right place.

For the record, I'd probably class the Windows subsystem for Linux as the Linux platform, not Windows.